packages/backend/src/ee/controllers/snowflakeController.ts

```diff
@@ -26,6 +34,13 @@ export class SnowflakeController extends BaseController {
         @Request() req: express.Request,
     ): Promise<ApiSuccessEmpty> {
         this.setStatus(200);
+
+        if (!lightdashConfig.license.licenseKey) {
```

Comment:
We will return an error, this will not prevent the user from login, this only lets you know if you are authenticated in the UI